### ISSUE TITLE
Conjur issue with command line and docker

### ISSUE THREAD

**Issue by saicharan9440** [cyberark/conjur#515]
Why am I getting this error? how do we fix this?

root@272c02832f2c:~/conjur/tutorials/secrets# conjur policy load --as-group security_admin --namespace $NAMESPACE policy.yml
error: Unknown option --as-group

NAME
    load - Load a policy

SYNOPSIS
    conjur [global options] policy load [command options] POLICY FILENAME

COMMAND OPTIONS
    --[no-]delete  - Allow explicit deletion statements in the policy.
    --[no-]replace - Fully replace the existing policy, deleting any data that
                     is not declared in the new policy.
root@272c02832f2c:~/conjur/tutorials/secrets#

**Comment by saicharan9440** [comment]
Anyone can help me on this issue ??

**Comment by kgilpin** [comment]
For this version of the command line interface please see the documentation at conjur.org

developer.conjur.net is documentation for the Conjur enterprise edition version 4

**Comment by saicharan9440** [comment]
Yeah, I did. But I couldn't get the proper answer. This is the error that I am facing now. If you know how to resolve then please let me know.
root@272c02832f2c:~/conjur/tutorials/secrets# conjur policy load --as-group security_admin --namespace $NAMESPACE policy.yml
error: Unknown option --as-group

**Comment by kgilpin** [comment]
I believe this tutorial is from developer.conjur.net which is for the v4 enterprise version of conjur. 

For tutorials on this open source Conjur 5 see  https://www.conjur.org/tutorials/
